CHANGELOG.md

```diff
@@ -44,6 +45,42 @@ All notable changes to this project will be documented in this file. This projec
     intl.formatMessage(messages.saveUnsavedChanges);
     ```
 
+-   class-transformer changes:
```

Comment:
Do we need to document class-transformer breaking changes? Aren't they documented in their CHANGELOG already?

Reply:
only ClassConstructor is not documented. I documented these changes here for easier comet migrations i can remove if you want?

Reply:
imho it is a good a idea to give a short overview here.

Reply:
(this is not an "Incompatible Change" though, as we didn't have a stable release so far)

Reply:
Okay, I'd leave it as it is. This changelog is good for our developers switching from one canary version to the next. We should remove it before the stable release though